src/test/java/org/broadinstitute/hellbender/utils/read/markduplicates/MarkDuplicatesSparkTester.java

```diff
@@ -9,6 +10,10 @@
  */
 public final class MarkDuplicatesSparkTester extends AbstractMarkDuplicatesTester {
 
+    public MarkDuplicatesSparkTester() {
+        super(true, DuplicateScoringStrategy.ScoringStrategy.TOTAL_MAPPED_REFERENCE_LENGTH);
```

Comment:
Now that you've changed the default, we definitely need tests for `SUM_OF_BASE_QUALITIES`. Either add them here, or create a ticket to add them after alpha.